cross-modal-search/requirements.txt

```diff
@@ -1,7 +1,7 @@
-jina[hub, http]==2.0.0rc5
+jina[hub, http]==2.0.4
```

Comment:
This better be `~=2.0.0`

Reply:
```suggestion
jina[hub, http]~=2.0.0
```

Reply:
test

Reply:
test2

Reply:
Addressed, please see [0e3acad](https://github.com/jina-ai/examples/pull/670/commits/0e3acad28fc311ec836f65e99209d5f1b84de393) for the change.